R/magora_boxplot.R

```diff
@@ -1,17 +1,39 @@
 #' Boxplot for phenotype data
 #'
 #' @param data Input data (\code{\link{pathology}} filtered by phenotype, mouse model, and tissue, with mouse models expanded via \code{\link{expand_mouse_model_factor_from_selection}}).
-#' @inheritParams magora_volcano_plot
+#' @param mouse_model_groups Mouse models
 #' @param use_theme_sage Whether to use \code{\link[sagethemes]{theme_sage}}. Defaults to TRUE.
 #'
 #' @return A ggplot2 object.
 #' @export
-magora_boxplot <- function(data, type = "ggplot2", facet = TRUE, save_name, use_theme_sage = TRUE) {
+magora_boxplot <- function(data, mouse_model_groups, use_theme_sage = TRUE) {
 
-  # Check arguments
-  if (!type %in% c("ggplot2", "plotly")) {
-    stop("`type` should be one of 'ggplot2' or 'plotly'", call. = FALSE)
-  }
+  phenotype_units <- unique(data[["phenotype_units"]])
```

Comment:
This seems like more pandas style.  dply selects?

Reply:
Hi Sharla, I'm trying to stay away from opinion, the tidyverse is full of it for better or worse.  I prefer to not mix base R and tidyverse when I can, with preference given to the tidyverse.  You may have a take, too.